charts/cryostat/templates/cryostat_deployment.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "cryostat.fullname" . }}
+  name: {{ include "cryostat.fullname" . }}-main
```

Comment:
I added this change so that `ct` can detect changes and run tests for this PR :)) 

If possible, I think it could be a temporary fix until #200.  Preferably, we might want to avoid using '--force' in upgrades.